app/services/csv/sales_log_csv_service.rb

```diff
@@ -129,31 +131,38 @@ def label_if_boolean_value(value)
       return "No" if value == false
     end
 
```

Comment:
In general I am wondering if there is value in quite heavily commenting the CSV export services. Almost everything they do is quite arbitrary, so I think it may reduce developer time very significantly in the future, particularly when it's being worked on by people with little to no context, to have a comment such as:
```
After getting all the question ids in order attribute_mappings are used to change the CSV headings while keeping the ordering from the form, for example adding derived fields nex tot he field they are derived from
```

Reply:
I'm not the biggest fan of comments, unless there's no other way to make the code readable, but there are probably some nicely named methods we can extract here etc.